src/tribler/gui/defs.py

```diff
@@ -210,6 +210,11 @@ def get(cls, item, default=None):
 
 
 UPGRADE_CANCELLED_ERROR_TITLE = "Tribler Upgrade cancelled"
+
 NO_DISK_SPACE_ERROR_MESSAGE = "Not enough storage space available. \n" \
                               "Tribler requires at least %s space to continue. \n\n" \
                               "Please free up the required space and re-run Tribler. "
+
+DATABASE_IS_CORRUPTED_ERROR_MESSAGE = "The database file of the previous version found to be corrupted: \n\n" \
```

Comment:
It seems to be nearly identical with the strings https://github.com/Tribler/tribler/pull/7628/files#diff-df47295d04e11b939d3d1756c77296b153338b035e33de997902ec673ad37043R1261-R1264

Perhaps it is worth extracting it to a common place.

Reply:
Indeed, I extracted it to a single place. Now it is possible because the simplified message box never requires actions from the user.